tests/integration/models/billing.spec.coffee

```diff
@@ -0,0 +1,257 @@
+			m.chai.expect(promise).to.become({
+				title: 'Free'
+				name: 'Free plan'
+				code: 'free'
+				tier: 'free'
+				billing:
+					currency: 'USD'
+					charges: [
+						{
+							itemType: 'plan'
+							name: 'Free plan'
+							code: 'free'
+							unitCostCents: 0
+							quantity: 1
+						}
+						{
+							itemType: 'support'
+							name: 'Community support'
+							unitCostCents: 0
+							quantity: 1
+						}
+					]
+					totalCostCents: 0
+				support:
+					title: 'Community'
+					name: 'Community support'
+			})
+
+		givenABillingAccountIt 'should return a paid tier billing plan object', ->
+			resin.models.billing.getPlan()
+			.then (plan) ->
+
+				m.chai.expect(_.pick(plan, [
+					'title'
+					'name'
+					'code'
+					'tier'
+					'intervalUnit'
+					'intervalLength'
+				])).to.deep.equal({
+					title: 'TestDev'
+					name: 'TestDev plan'
+					code: 'testdev'
+					tier: 'testdev'
+					intervalUnit: 'months'
+					intervalLength: '1'
+				})
+
+				m.chai.expect(plan).to.have.property('currentPeriodEndDate').that.is.a('string')
+				m.chai.expect(plan).to.have.property('description').that.is.a('string')
+
+				m.chai.expect(plan).to.have.property('billing').that.deep.equals({
+					currency: 'USD'
+					charges: [
+						{
+							itemType: 'plan'
+							name: 'TestDev plan'
+							code: 'testdev'
+							unitCostCents: 1
+							quantity: 1
+						}
+						{
+							itemType: 'support'
+							name: 'Standard support'
+							unitCostCents: 0
+							quantity: 1
+						}
+					]
+					totalCostCents: 1
+				})
+				m.chai.expect(plan).to.have.property('support').that.deep.equals({
+					title: 'Standard'
+					name: 'Standard support'
+				})
+
+	describe 'resin.models.billing.getBillingInfo()', ->
+		withoutABillingAccountIt 'should return a free tier billing info object', ->
+			promise = resin.models.billing.getBillingInfo()
+			m.chai.expect(promise).to.become({})
+
+		givenABillingAccountIt 'should return a billing info object', ->
+			resin.models.billing.getBillingInfo()
+			.then (billingInfo) ->
+				m.chai.expect(billingInfo).to.not.be.null
+				if billingInfo.card_type == 'Visa'
+					m.chai.expect(billingInfo).to.deep.equal({
+						first_name: 'John'
+						last_name: 'Doe'
+						company: 'Resin.io'
+						vat_number: 'GBUK00000000000'
+						address1: 'One London Wall'
+						address2: '6th floor'
+						city: 'London'
+						state: 'Greater London'
+						zip: 'EC2Y 5EB'
+						country: 'GB'
+						phone: '6970000000'
+						card_type: 'Visa'
+						last_four: '1111'
+						type: 'credit_card'
+						first_one: '4'
+						year: '2018'
+						month: '8'
+						full_name: 'John Doe'
+					})
+				else
+					m.chai.expect(billingInfo).to.deep.equal({})
+
+	describe 'resin.models.billing.updateBillingInfo()', ->
+		# givenABillingAccountIt 'should return an billing account info object', ->
+		# 	promise = resin.models.billing.updateBillingInfo('')
+		# 	m.chai.expect(promise).to.become({})
```

Comment:
Removing it. I think that a positive test isn't possible since it would require `recurly.js` client library + the generated tokens are only valid for 20 minutes.